inception-project/inception
# #1200 - Selected Document Annotation not visible in details

**What's in the PR**
- document annotation side panel now works as an accordion instead of two separate panels
- the accordion only shows details for the currently selected annotation

**How to test manually**
* open a document in the brat or pdf editor
* create document annotations
* see if everything still works (create, select, delete, update values)

**Automatic testing**

**Documentation**
